package.json

```diff
@@ -8,14 +8,14 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/Banno/polymer-webpack-loader.git"
+    "url": "git+https://github.com/webpack-contrib/polymer-webpack-loader.git"
   },
   "author": "bryandcoulter@gmail.com",
   "license": "Apache-2.0",
```

Comment:
So far the instructions I have explicitly stated to leave the license field in package.json alone.